lightning/src/ln/channelmanager.rs

```diff
@@ -4568,13 +4614,60 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 	/// [`PaymentReceived`]: events::Event::PaymentReceived
 	/// [`PaymentReceived::payment_preimage`]: events::Event::PaymentReceived::payment_preimage
 	/// [`create_inbound_payment_for_hash`]: Self::create_inbound_payment_for_hash
+	// LDK does not store any data for pending inbound payments. Instead, in the case of
```

Comment:
Probably worth putting part of this, but probably not the actual algorithm, in the docs, no?